analyses/molecular-subtyping-integrate/01-integrate-subtyping.Rmd

```diff
@@ -0,0 +1,196 @@
+---
+title: "Integrate molecular subtyping results"
+output: 
+  html_notebook:
+    toc: true
+    toc_float: true
+author: Krutika Gaonkar for D3b
+date: 2020
+params:
+   is_ci: FALSE
+---
+
+The purpose of this notebook is to integrate molecular subtyping results from 
+[molecular-subtyping-pathology](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/molecular-subtyping-pathology) with `pbta-histologies-base.tsv`.
+
+Here we are using v18 `pbta-histologies-base.tsv`
+
+
+## Set up
+
+```{r}
+library(tidyverse)
+data_dir <- "../../data/release-v18-20201123/"
+base_histology <- read_tsv(file.path(data_dir,"pbta-histologies-base.tsv"),
+                           col_types = readr::cols(molecular_subtype = readr::col_character(),
+                                                   short_histology = readr::col_character(),
+                           broad_histology = readr::col_character(),
+                           Notes = readr::col_character())) %>%
+  unique()
+```
+
+### Read molecular-subtyping-pathology results
+
+Reading molecular_subtype,	integrated_diagnosis, short_histology,	broad_histology and	Notes from `compiled_molecular_subtypes_with_clinical_pathology_feedback.tsv` 
+
+```{r}
+
+compiled_subtyping<-read_tsv("../molecular-subtyping-pathology/results/compiled_molecular_subtypes_with_clinical_pathology_feedback.tsv")
+
+```
+
+Update "Other" sample broad/short histology and harmonized_diagnosis and add to `compiled_subtyping`
+
+```{r}
+
+Other_subtypes <- read_tsv(file.path("input","pathology_dx and pathology_free_text_diagnosis to broad_histology for subtyping module - rules_without_subtype_WIP.tsv"))
```

Comment:
I'm not a fan of this file name but I'm guessing it comes from somewhere else and we don't have control over that.

Reply:
umm I could update the name and add the link to the googlesheet tab used, just wanted to keep the name as is in the google sheet for reference.

Reply:
I think the file name should be changed everywhere or not at all. But if changing it will cause confusion, don't bother, no worries.